bedwars-plugin/src/main/java/com/tomkeuper/bedwars/sidebar/BoardManager.java

```diff
@@ -459,37 +460,40 @@ private void setHeaderFooter(TabPlayer player, IArena arena) {
         if (arena.isSpectator((Player) player.getPlayer())) {
             TabAPI.getInstance().getHeaderFooterManager().setHeaderAndFooter(
                     player, lang.m(Messages.FORMATTING_SIDEBAR_TAB_HEADER_SPECTATOR),
-                    lang.m(Messages.FORMATTING_SIDEBAR_TAB_FOOTER_SPECTATOR)
+                    lang.m(Messages.FORMATTING_SIDEBAR_TAB_FOOTER_SPECTATOR).replace("%bw_kills%", String.valueOf(arena.getPlayerKills(((Player) player.getPlayer()).getPlayer(), false))).replace("%bw_final_kills%", String.valueOf(arena.getPlayerKills(((Player) player.getPlayer()).getPlayer(), true))).replace("%bw_beds%", String.valueOf(arena.getPlayerBedsDestroyed(((Player) player.getPlayer()).getPlayer())))
             );
             return;
         }
 
-        String headerPath = null;
-        String footerPath = null;
+        Bukkit.getServer().getScheduler().scheduleSyncRepeatingTask(plugin, () -> {
```

Comment:
Why?

Reply:
> Why?

I need a repeat task to update this placeholder. But I haven't been able to fully fix it yet.